packages/patternfly-4/react-core/src/components/Alert/Alert.d.ts

```diff
@@ -8,9 +8,10 @@ export const AlertVariant: {
   info: 'info';
 };
 
-export interface AlertProps extends HTMLProps<HTMLDivElement> {
+export interface AlertProps extends Omit<HTMLProps<HTMLDivElement>, 'action'> {
   variant: OneOf<typeof AlertVariant, keyof typeof AlertVariant>;
   children?: ReactNode;
+  action?: ReactNode;
```

Comment:
Do you mind adding the other missing props? title, className, 'aria-label' i believe

Reply:
I didn't add `className`, `title` and `arira-label` because they are coming from `HTMLProps<HTMLDivElement>`.
Should I add them, though?

Reply:
ah that sounds good then, should be fine I think